javascript/0394-decode-string.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Stack
+ * https://leetcode.com/problems/decode-string/
+ * 
+ * Time O(n) | Space O(n)
+ * @param {string} s
+ * @return {string}
+ */
+var decodeString = function(s) {
+  
+const myStack = [];
+let result = '';
+for(let i = 0; i < s.length; i++) {
+    if(s[i] !== ']') {
+        myStack.push(s[i]);
+    } else {
```

Comment:
Suggestion: avoid else

```js
if (true) continue;

...
```

Reply:
Getting rid of the else case. You truly have some animosity with the **else**  😅 😅 😅. I have removed it.

Reply:
Just a kind reminder that I updated the code as suggested.